### ISSUE TITLE
EXL2 quants at 4.65 bits in dual 3090 gpu´s

### ISSUE THREAD

**Issue by jostack** [turboderp/exllamav2#91]
I have 2 RTX 3090s with NVLink, 128 GB of DDR4 RAM, and a Ryzen 9 5950x. Will I be able to run the Llama2-70B model with EXL2 quants at 4.65 bits per weight using this setup? Additionally, does Exllama V2 support dual GPU configurations?

**Comment by turboderp** [comment]
Yes, dual GPUs are supported. I run 70B at 4.65 bits on a 4090 plus a 3090Ti (also 24 GB). You should even be able to go slightly higher bitrate. You'll want a GPU split of something like (19,24), but YMMV a bit on that.

**Comment by jostack** [comment]
> Yes, dual GPUs are supported. I run 70B at 4.65 bits on a 4090 plus a 3090Ti (also 24 GB). You should even be able to go slightly higher bitrate. You'll want a GPU split of something like (19,24), but YMMV a bit on that.

Hello @turboderp 

First and foremost, I want to express my sincere gratitude for taking the time to address my inquiry. I must seize this opportunity to congratulate you on the impressive project you're working on and for everything you've achieved so far. The passion and commitment you invest in every detail are evident.

Repositories and resources like yours are essential for those of us who are just starting out in this field. Your contribution allows us to accelerate our learning and progress with confidence.

I'm eager to implement what I've learned in my own environment. Moreover, I'm exploring how I can make the most of the new nvlink connection I've recently acquired.

neuron@neuron:~$ nvidia-smi topo -m
        GPU0    GPU1    CPU Affinity    NUMA Affinity   GPU NUMA ID
GPU0     X      NV4                             N/A
GPU1    NV4      X                              N/A


 Of course, my intention is to provide as much support as possible to the project, whether through feedback, improvement proposals, or any other way I can add value.

I want to reiterate my thanks, and I hope we can stay in touch.

Warm regards,
